abp168/index.rst

```diff
@@ -0,0 +1,29 @@
+.. template documentation master file, created by
```

Comment:
Delete this file.

Reply:
Just to clarify, you want me to delete the index.rst file entirely? I wouldn't have a rest file anymore if so.

Reply:
Deleted